reana_job_controller/kubernetes_job_manager.py

```diff
@@ -102,6 +103,9 @@ def execute(self):
                                 'name': backend_job_id,
                                 'env': [],
                                 'volumeMounts': [],
+                                'resources': {
+                                    "limits": {"memory": "2Gi"}
```

Comment:
Let's extract it to a configuration variable as in: https://github.com/reanahub/reana-job-controller/blob/2a7987954e7d9d90d8edbdd79b103873dc4ba946/reana_job_controller/config.py#L63